schemas/questions/types/relationship.json

```diff
@@ -1,57 +0,0 @@
-{
```

Comment:
How come this has been removed? Is it a type of repeat?

Reply:
Its also been removed from runner